openstudiocore/resources/model/OpenStudio.idd

```diff
@@ -21801,14 +21801,14 @@ OS:Sizing:System,
        \object-list AirPrimaryLoops
   A3, \field Type of Load to Size On
        \note Specifies the basis for sizing the system supply air flow rate
-       \note Sensible and VentilationRequirement are the only available options
```

Comment:
@jmarrec I merged develop into develop3 but this is still showing as a diff.  Are there some other branches mixed in here?  I saw some more port/const stuff below as well.  All that should get merged to develop and develop3.  I wonder if there are some other PRs I need to merge to develop?